From 24d8e9ac3000d94b4a519630c189dd6145a1d9dd Mon Sep 17 00:00:00 2001
From: wangtao12 <wangtao12@oppo.com>
Date: Sat, 17 Jan 2026 11:16:46 +0800
Subject: [PATCH 086/223] ANDROID: GKI: Export memcg functions to allow module
 to add new files

1. Since memcg version has changed from v1 to v2, it's necessry to export
symbol cgroup_add_dfl_cftypes for developing features.

1 function symbol(s) added
  'int cgroup_add_dfl_cftypes(struct cgroup_subsys*, struct cftype*)'

Bug: 417133503
Bug: 500888456
Bug: 503656770

Signed-off-by: wangtao12 <wangtao12@oppo.com>
Change-Id: I9b83aae1283d4ffbb2d7bcca7dfca5ab0437f80e
Signed-off-by: wangtao12 <wangtao12@oppo.com>
(cherry picked from commit 7fc94e622fcee5e043c9ce07cd31be6c92241a9e)
(cherry picked from commit 6afa8a82eb55af945c924c3e28c92971ebbaf434)
(cherry picked from commit 2f21478a6a269e2f1075b0a15e76a26a8a4ce3c4)
---
 gki/aarch64/symbols/oplus | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gki/aarch64/symbols/oplus b/gki/aarch64/symbols/oplus
index 926ee460c226..81b67936485b 100644
--- a/gki/aarch64/symbols/oplus
+++ b/gki/aarch64/symbols/oplus
@@ -136,6 +136,7 @@
   mem_cgroup_update_lru_size
   memory_cgrp_subsys
   memory_cgrp_subsys_enabled_key
+  cgroup_add_dfl_cftypes
   mipi_dsi_generic_write
   mmc_wait_for_cmd
   mnt_drop_write
-- 
2.50.1

